CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.41.1
+* Bug - Change ports response from Ports to KnownPortBindings when present HostPort of taskMetadata endpoint response [#2495](https://github.com/aws/amazon-ecs-agent/pull/2495)
```

Comment:
nit: it is a bit unclear what problem we've addressed with this message. IMO we can refer to original bug report issue https://github.com/aws/amazon-ecs-agent/issues/2476 and use this changelog:
Fixed a bug where HostPort is not present in ECS Task Metadata Endpoint response with bridge network type [#2495](https://github.com/aws/amazon-ecs-agent/pull/2495)

Reply:
sure, will change it.